### PR TITLE
WebView -> WKWebView (for SPTooltip only)

### PR DESCRIPTION
**Pull request by jamesstout**
What does this implement/fix? Explain your changes.
---------------------------------------------------
Migrate to WKWebView

Any other comments?
-------------------
See: https://developer.apple.com/documentation/webkit/webview

Where has this been tested?
---------------------------
**Devices/Simulators:** iMac19,1

**macOS Version:**: 10.15.7 (19H2)

**Sequel-Ace Version:** arc_updates latest

### CODE REVIEW

File: Source/Model/SPTooltip.h [hunk 1]
```diff
@@ -43,6 +44,10 @@
 	NSPoint			mousePositionWhenOpened;
 	
 	NSString* 		SPTooltipPreferencesIdentifier;
+	
+	BOOL 			gotHeight;
```
Comment (Kaspik) [inline Source/Model/SPTooltip.h:48]: Can we start using `@property (nonatomic, assign) BOOL gotHeight;` inside `m` file so we have the atomics and assign already in for ARC? :)

**Review by Kaspik — APPROVED** [review]
Oh I wanted to do this so badly, thanks for doing it!

File: Source/Model/SPTooltip.h (hunk 1 above)
Comment (jamesstout) [reply Source/Model/SPTooltip.h:48]: will do

**Review by Kaspik — APPROVED** [review]

File: Source/Model/SPTooltip.h [hunk 2]
```diff
@@ -30,9 +30,10 @@
 
 #import <WebKit/WebKit.h>
```
Comment (Jason-Morcos) [inline Source/Model/SPTooltip.h:31]: I know this has already been merged, but can we drop the webkit import here when we switch to WKWebView?

Comment (jamesstout) [reply Source/Model/SPTooltip.h:31]: It's still part of [WebKit](https://developer.apple.com/documentation/webkit).

**Pull request merged**